### PR TITLE
removed dead code (part 8)

### PR DESCRIPTION
**Pull request by ChristianKienle**

### CODE REVIEW

**Review by steins-code — CHANGES_REQUESTED** [review]
Please when you delete app strings also delete corresponding entries in strings files en and de

**Comment by ChristianKienle** [comment]
> Please when you delete app strings also delete corresponding entries in strings files en and de

🥺 shiii forgot.

**Comment by ChristianKienle** [comment]
> Please when you delete app strings also delete corresponding entries in strings files en and de

done

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 1]
```diff
@@ -432,31 +415,11 @@ enum AppStrings {
...
-		static let resultCardPositiveTitle = NSLocalizedString("Home_resultCard_PositiveTitle", comment: "")
-		static let resultCardPositiveDesc = NSLocalizedString("Home_resultCard_PositiveDesc", comment: "")
-		static let resultCardPendingTitle = NSLocalizedString("Home_resultCard_PendingTitle", comment: "")
 		static let resultCardPendingDesc = NSLocalizedString("Home_resultCard_PendingDesc", comment: "")
 		static let resultCardInvalidTitle = NSLocalizedString("Home_resultCard_InvalidTitle", comment: "")
 		static let resultCardInvalidDesc = NSLocalizedString("Home_resultCard_InvalidDesc", comment: "")
 	}
-
-	enum RiskView {
-		static let unknownRisk = NSLocalizedString("unknown_risk", comment: "")
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:444]: these RiskView strings still are in the Localizable files

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 2]
```diff
@@ -432,31 +415,11 @@ enum AppStrings {
 		static let resultCardShowResultButton = NSLocalizedString("Home_resultCard_ShowResultButton", comment: "")
 		static let resultCardNegativeTitle = NSLocalizedString("Home_resultCard_NegativeTitle", comment: "")
 		static let resultCardNegativeDesc = NSLocalizedString("Home_resultCard_NegativeDesc", comment: "")
-		static let resultCardPositiveTitle = NSLocalizedString("Home_resultCard_PositiveTitle", comment: "")
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:435]: these also remain :)

Comment (ChristianKienle) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:435]: done.

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 1 above)
Comment (ChristianKienle) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:444]: done

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 3]
```diff
@@ -432,31 +415,11 @@ enum AppStrings {
...
-		static let resultCardPositiveDesc = NSLocalizedString("Home_resultCard_PositiveDesc", comment: "")
-		static let resultCardPendingTitle = NSLocalizedString("Home_resultCard_PendingTitle", comment: "")
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:437]: this remains in localizable

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 4]
```diff
@@ -432,31 +415,11 @@ enum AppStrings {
 		static let resultCardShowResultButton = NSLocalizedString("Home_resultCard_ShowResultButton", comment: "")
 		static let resultCardNegativeTitle = NSLocalizedString("Home_resultCard_NegativeTitle", comment: "")
 		static let resultCardNegativeDesc = NSLocalizedString("Home_resultCard_NegativeDesc", comment: "")
-		static let resultCardPositiveTitle = NSLocalizedString("Home_resultCard_PositiveTitle", comment: "")
-		static let resultCardPositiveDesc = NSLocalizedString("Home_resultCard_PositiveDesc", comment: "")
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:436]: this remains too in de and en

**Review by haosap — APPROVED** [review]

**Review by steins-code — APPROVED** [review]

**Pull request merged**